packages/server/src/app/app.service.ts

```
@@ -184,4 +191,26 @@ export class AppService {
     hmac.update(input);
     return hmac.digest('hex');
   }
+
+  private async updateUserBalance(supporter: SupporterDocument, pledgeAmount: number, totalContribution: number): Promise<any> {
+    const capStr = process.env.PATREON_TOKEN_BALANCE_CAP
+    if (pledgeAmount && totalContribution && capStr) {
+      const cap = parseInt(capStr);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)